tests/integration/users/test_views.py

```diff
@@ -12,7 +12,9 @@ def test_get_anonymous(self):
         response = self.client.get(self.url)
 
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(response.content, b'{"user_id":null}')
+        self.assertEqual(
+            json.loads(response.content.decode('utf8')),
```

Comment:
Here we get the "Quittung". In Python 3.3 and 3.4 the result is `b'{"user_id":null,"guest_enabled":false}'` but in Python 3.5 it is `b'{"guest_enabled":false,"user_id":null}'` for whatever reason, so this test always failed. We should not use bytestring tests any more if we want to check JSON content.